app/modules/organizations/models.py

```diff
@@ -5,25 +5,124 @@
+TIMEZONE = pytz.timezone('US/Pacific')
+DATETIME_FMTSTR = '%Y-%m-%dT%H:%M:%S.%fZ'
 
-class Organization(db.Model, HoustonModel):
+log = logging.getLogger(__name__)
+
+
+class OrganizationEDMMixin(EDMObjectMixin):
+
+    # fmt: off
+    # Name of the module, used for knowing what to sync i.e organization.list, organization.data
+    EDM_NAME = 'organization'
+
+    # The EDM attribute for the version, if reported
+    EDM_VERSION_ATTRIBUTE = 'version'
+
+    #
+    EDM_LOG_ATTRIBUTES = [
+        'name',
+    ]
+
+    # todo Very much a first drop of mapping
+    EDM_ATTRIBUTE_MAPPING = {
+        # Ignored
+        'id'                    : None,
+        'created'               : None,
+        'modified'              : None,
+
+        # # Attributes
+        'name'                  : 'title',
+        'url'                   : 'website',
+        'version'               : 'version',
+
+        # # Functions
+        'members'               : '_process_members',
+        'logo'                  : '_process_logo',
+        'createdDate'           : '_process_created_date',
+        'modifiedDate'          : '_process_modified_date',
+    }
+    # fmt: on
+
+    @classmethod
+    def ensure_edm_obj(cls, guid):
+        organization = Organization.query.filter(Organization.guid == guid).first()
+        is_new = False
+
+        if organization is None:
+            organization = Organization(
+                guid=guid,
+                title='none',
+            )
+            db.session.add(organization)
+            is_new = True
+
+        return organization, is_new
+
+    def _process_members(self, members):
+        for member in members:
+            log.info('Adding Member ID %s' % (member.id,))
+            user, is_new = User.ensure_edm_obj(member.id)
+
+            # todo restore once the tests work
+            # with db.session.begin():
+            #    self.members.append(user)
+
+    def _process_logo(self, logo):
+        self.logoGuid = logo.uuid
+        self.logoUrl = logo.url
+
+    def _process_created_date(self, created_date):
+        self.createdDate = datetime.datetime.strptime(
+            created_date, DATETIME_FMTSTR
+        ).astimezone(TIMEZONE)
+
+    def _process_modified_date(self, modified_date):
+        self.modifiedDate = datetime.datetime.strptime(
+            modified_date, DATETIME_FMTSTR
+        ).astimezone(TIMEZONE)
+
+
+class Organization(db.Model, HoustonModel, OrganizationEDMMixin):
     """
     Organizations database model.
     """
 
     guid = db.Column(
         db.GUID, default=uuid.uuid4, primary_key=True
     )  # pylint: disable=invalid-name
+    version = db.Column(db.Integer, default=None, nullable=True)
     title = db.Column(db.String(length=50), nullable=False)
+    # todo why can dateTimes in the auth class be non Nullable but in here the upgrade fails?
+    createdDate = db.Column(
```

Comment:
A `HoustonModel` inherits from `FeatherModel` which has a parent `TimestampViewed` which inherits from `Timestamp`.  The models `TimestampViewed` and `Timestamp` create the database columns `viewed` and `created`, respectively.  These columns can be deleted since it is implicit.